dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -492,6 +504,29 @@ export default function CollectCardPaymentScreen() {
         </Picker>
       </List>
 
+      {discoveryMethod === 'internet' && (
```

Comment:
routing priority should be visible for all discovery methods, not just internet

Reply:
got it~.